docs/section6/Rule6-4.md

```diff
@@ -8,11 +8,17 @@
 
 **Applicability:** All required data elements exist for P_RMR  
 **Applicability Checks:** None  
-**Manual Check:** None  
+**Manual Check:** 
+
+1. Where a complete lighting system exists, the actual lighting power cannot be verified by the RCT. Manual review is required.  
+
```

Comment:
@xingcx could you please remind me the significance of the Manual Check? How does it address the issue of RCT not being able to tell whether lighting is newly designed, existing to remain or not designed? Since we don't have the attribute(s) in the RMR that would allow us to differentiate between these three scenario, should all three be covered by the same RDS? 

This decision would apply to all building systems and components, not just lighting. For example, for envelope G3.1 #5a says "a. All components of the building envelope in the proposed design shall be modeled as shown on architectural drawings or as built for existing building envelopes."  Also, Table G3.1 #1c says
c. When the Performance Rating Method is applied to buildings in
which energy-related features have not yet been designed
(e.g., a lighting system), those yet-to-be-designed features
shall be modeled in the proposed design to comply with but not
exceed the requirements of this standard as described in Table
G3.1, Nos. 6, 10, 11, and 12. 
I suggest that in all such cases RCT checks if U-RMR=P-RMR, and perhaps additionally check if P-RMR reflects minimum requirements of 90.1 Sections 5-10, as applicable. @supriyagoel please take a look also.

Reply:
@mkarpman and @xingcx the 3 RDS for as existing, as designed and not yet designed - would all require manual check.
Maybe we have a CAUTION outcome for when P-RMR = U-RMR and a FAIL outcome for when P-RMR is not equal to U-RMR. @mkarpman - would that work?
Xing, please have a look out here-
https://docs.google.com/document/d/1KNEe0Igcv8CA6e9u4rlKCCGsvAsNNvjN1_mmpPQeuQc/edit?usp=sharing

Reply:
@mkarpman its good to have 3 different RDS for this, even though the outcome is the same. This is because these are 3 different rules in 90.1  and we want these to be separate rule definitions with separate outcomes. Hope that makes sense

Reply:
@supriyagoel I don't mind having different RDSs for existing, as designed and not yet designed, but wouldn't they all be exactly the same? We may have the following scenarios:
1. U-RMR = P-RMR 
a. correct if component is existing or as designed
b. may or may not be correct if component is yet to be designed because U-RMR may use assumptions other than what is prescribed in App G for such scenarios.  

2. U-RMR != P-RMR
a. wrong if component is existing or as designed
b. may or may not be correct depending on whether P-RMR matches what is prescribed in App G for such scenarios.  

You suggest to show Caution for (1) and Fail for (2), but maybe it should be more nuanced. I think as James suggested, we should also check if P-RMR reflects BAM LPDs and if they do show caution + "P-RMR reflects yet-to-be designed default LPD" in cases 1b and 2b.

I think we will have similar situations with systems other than lighting as well.